bundles/org.openhab.binding.fritzboxtr064/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fritzboxtr064"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="rootdevice">
+		<label>Fritzbox Root-Device</label>
+		<description>A physical Fritzbox Device.</description>
+
+		<config-description>
+			<parameter name="host" type="text">
+				<label>Host</label>
+				<description>FQDN or IP of FritzBox.</description>
+				<default>fritz.box</default>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="user" type="text" required="true">
+				<label>Username</label>
+				<default></default>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="refresh" type="integer">
+				<label>Refresh Interval</label>
+				<default>60</default>
+				<unitLabel>s</unitLabel>
+			</parameter>
+			<parameter name="tamIndices" type="text" multiple="true">
```

Comment:
Hm, looking at your XMLs below, it seems that quite some of those parameters (and channels) are Fritzbox-specific and not TR-064 standard, is that correct?
It would imho thus make sense to define different "root device" types - we could have a "generic-cpe" and a "fritzbox" type, only the latter having all the Fritzbox details. Would that make sense?
Likewise, we could in future add a `speedport` thing type to also cover speedport-specific parameters/channels for those devices.